### DISCUSSION TITLE
Kasabın zararı nedir

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#220] [Zeka Sorusu]
Afacan Ali kasaba bir kilo et almaya gider, kasapta etin kilosu 9 TL’dir. Kasaba 1 kilo et istediğini söyler, kasap eti hazırlamak için arkasını döner, Afacan Ali o sırada tezgahın üzerinde duran 10 TL’yi görür ve cebine atar. Kasap Ali ye eti verir. Afacan Ali cebinden biraz önce almış olduğu 10 TL’yi kasaba uzatır, kasapta para üstü olarak 1 TL’yi öder.
Afacan Ali kasaptan sanki iyi bir iş yapmış gibi sırıtarak çıkar gider.
Dolandırılan kasabımız toplam bu alışverişten kaç lira zarar etmiştir?

**Comment by yukseel** [comment]
Önce 10 lira zarara giriyor fakat 10 lirayı bakkala geri veriyor burdan da 1TL para üstü verirken 1TL zarar oluyor,
1 kilo etin fiyatı da 9 tl olduğu için 1 kilo et fiyatı kadar da zarara giriyor çünkü bedavaya almış oluyor toplam 10 tl zarar
